connect4/__init__.py

```diff
@@ -31,4 +31,4 @@
 
 def setup(bot):
```

Comment:
need to make this async

Reply:
oh yeah forgot to do that, that's for addressing!